taxi/views.py

```diff
@@ -28,6 +30,22 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required()
+def car_driver_assignment(
+        request: HttpRequest,
+        pk: int,
+        driver_id: int
+) -> HttpResponse:
+    car = get_object_or_404(Car, pk=pk)
+    driver = get_object_or_404(Driver, pk=driver_id)
```

Comment:
you sure we need to check if we can get object with login_required decorator?)
also checklist.